pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -216,17 +216,23 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 
 	if failSwapOn {
 		// Check whether swap is enabled. The Kubelet does not support running with swap enabled.
-		swapData, err := ioutil.ReadFile("/proc/swaps")
+		swapFile := "/proc/swaps"
```

Comment:
You could maybe make this a constant and move it to the top of the file, but that is up to you.

Reply:
Thanks @odinuge !  Your suggestion is good! 
But since it is only used in this function, so I prefer to keep it that way for now.